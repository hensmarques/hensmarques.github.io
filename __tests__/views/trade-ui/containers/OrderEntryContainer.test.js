import React from 'react';
import OrderEntryContainer from '../../../../src/views/trade-ui/containers/OrderEntryContainer';

describe('<OrderEntryContainer/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<OrderEntryContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});
