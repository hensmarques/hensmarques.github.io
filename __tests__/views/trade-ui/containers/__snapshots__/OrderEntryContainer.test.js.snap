// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderEntryContainer/> renders and matches the snapshot 1`] = `
<WidgetWrapper
  classNames={
    Array [
      "order-entry",
    ]
  }
  tabs={
    Array [
      "Order Entry",
    ]
  }
>
  <Connect(OrderEntry) />
</WidgetWrapper>
`;
